docs/01_nodeos/06_logging/10_native_logging/index.md

```diff
@@ -0,0 +1,112 @@
+---
+content_title: Nodeos Native Logging
+link_text: Nodeos Native Logging
```

Comment:
link_text: Native Logging
(for consistency, as in the other left nav menu items under Nodeos)